ui/src/templates/reducers/index.ts

```diff
@@ -65,7 +65,17 @@ export const templatesReducer = (
       case SET_ACTIVE_COMMUNITY_TEMPLATE: {
         const {template} = action
 
-        draftState.activeCommunityTemplate = template
+        const activeCommunityTemplate: CommunityTemplate = {}
+        activeCommunityTemplate.dashboards = template.dashboards || []
+        activeCommunityTemplate.telegrafConfigs = template.telegrafConfigs || []
+        activeCommunityTemplate.buckets = template.buckets || []
+        activeCommunityTemplate.checks = template.checks || []
+        activeCommunityTemplate.variables = template.variables || []
+        activeCommunityTemplate.notificationRules =
+          template.notificationRules || []
+        activeCommunityTemplate.labels = template.labels || []
+
+        draftState.activeCommunityTemplate = activeCommunityTemplate
```

Comment:
I wonder if there's a way to handle this using a "default template" and a spread operator. More of a code style thing

Reply:
@alexpaxton haha that's funny, love where your head's at. I actually had it as

```ts
activeCommunityTemplate.dashboards = [...template.dashboards]
```
but these are set as `null` not `undefined` so spreading doesn't work.